regions/core/mask.py

```diff
@@ -99,10 +99,17 @@ def _overlap_slices(self, shape):
 
         return slices_large, slices_small
 
-    def _to_image_partial_overlap(self, image):
+    def to_image_partial_overlap(self, image):
```

Comment:
def _to_image_partial_overlap(self, image):

we do not necessary even want to change

Reply:
but u can do.

Reply:
I don't understand your comment.